plugins/redfish/fu-redfish-backend.c

```diff
@@ -118,8 +118,11 @@ fu_redfish_backend_coldplug_member(FuRedfishBackend *self, JsonObject *member, G
 			   NULL);
 
 	/* Dell specific currently */
-	if (self->system_id != NULL)
+	if (self->system_id != NULL) {
 		fu_device_add_instance_str(dev, "SYSTEMID", self->system_id);
+		/* This ensure the reboot is not done immediately after installation, and only after a wanted reboot */
```

Comment:
no need for `This...`  just launch into `ensure the reboot is not done immediately after installation, and only after a wanted reboot`